apps/web/src/components/forms/signup.tsx

```diff
@@ -21,7 +21,7 @@ import { useToast } from '@documenso/ui/primitives/use-toast';
 export const ZSignUpFormSchema = z.object({
   name: z.string().trim().min(1, { message: 'Please enter a valid name.' }),
   email: z.string().email().min(1),
-  password: z.string().min(6).max(72),
+  password: z.string().min(6, { message: 'Password should contain at least 6 characters' }).max(72),
```

Comment:
Since it has min and max, should the message be something like "Password should contain between 6 and 72 characters"?

Reply:
Generallly we don't set password more then 72 characters. So, I think it's fine now.
What if we can add validation messages for both the min and max cases? For max- "Password should not contain more than 72 characters".

Reply:
@catalinpit what do you think?

Reply:
You are right. Please add the message for the max value as well.

Reply:
Done